addressbook.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto2";
+
+package tutorial;
+
+option java_package = "com.example.tutorial";
+option java_outer_classname = "AddressBookProtos";
```

Comment:
Maybe don't need this.
Adding it will make Person a nested class within AddressBookProtos.
Without it Person will be a top level class.

Reply:
If I remove `option java_outer_classname = "AddressBookProtos";`, it seems it will still create a outer class by default